g2p/mappings/langs/str/str_equiv.json

```diff
@@ -0,0 +1,78 @@
+[
+    {
+        "in": "Á",
+        "out": "A\u0301"
+    },
+    {
+        "in": "Ⱥ",
+        "out": "Ⱥ"
+    },
+    {
+        "in": "Ć",
+        "out": "C\u0301"
```

Comment:
Some of your rules are going from NFC to NFD (like A or C), and some are going from NFD to NFC (like I or S) - why is that? Is that what the keyboards do?

Reply:
yeah, I just kept the NFC/NFD configuration of the output characters as they already were in str-to-ipa, so this sometimes meant I was going to NFC and sometimes to NFD. Generally the letters like Ⱥ were NFD originally, and some of the ones like Í were NFC.

Reply:
Interesting - is that confirmed from anything? Like, are we sure str-to-ipa is correct?

Reply:
Yes, I think so, because in both the SENĆOŦEN keyboards I'm using, the characters arent consistently NFC or NFD. theyre mostly NFC, but W̱ and X̱ are always NFD. However I double-checked now and noticed some rules that would normally be in the equiv are in the ipa-mapping itself (e.g. the rules on line7 and line11), so i deleted the redundancy in the equiv just in my sandbox and it runs alright on my end. Should i push that change, or should i move the rules out of str-ipa into the equiv?

Reply:
> Yes, I think so, because in both the SENĆOŦEN keyboards I'm using, the characters arent consistently NFC or NFD. theyre mostly NFC, but W̱ and X̱ are always NFD. However I double-checked now and noticed some rules that would normally be in the equiv are in the ipa-mapping itself (e.g. the rules on line7 and line11), so i deleted the redundancy in the equiv just in my sandbox and it runs alright on my end. Should i push that change, or should i move the rules out of str-ipa into the equiv?

Good question - I think it would actually be best to move the normalization/equivalency parts of the ipa mapping to the equiv mapping, just to be more consistent with the purpose of each mapping. Can you do that and push the change? I'll merge it after that. Thanks @shankhalika-katya !

Reply:
done! thanks @roedoejet !

Reply:
That was fast! Thanks @shankhalika-katya !